app/models/fullcalendar_engine/event_series.rb

```diff
@@ -1,7 +1,7 @@
 module FullcalendarEngine
   class EventSeries < ActiveRecord::Base
     
-    attr_accessor :title, :description, :commit_button
+    attr_accessor :title, :description, :commit_button, :color
```

Comment:
Ah, good point. My intention is to add this functionality later on. Right now I'm just setting the color when the object in the controller. I never set the color attribute from the GUI. I should probably be added before merging this pull request, right?